ideprobe/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM openjdk:11
+FROM openjdk:8
```

Comment:
Isn't it better to keep 11?

Reply:
Pants plugin is built on java 8 so I just want to make it safer in terms of compatibility

Reply:
@marek1840 didn't we need to switch to 11 since 2020.1?

Reply:
If I recall correctly ww were using 11 (only in tests) when the Scala plugin was built using 11. Is it no longer the case?

Reply:
Yi mentioned here that it is on java 8 https://github.com/pantsbuild/intellij-pants-plugin/pull/540

Reply:
Yeah the plugin is built with 8. However the runtime used for IntelliJ is 11. So either version is fine for now. 

It would be good to explore whether we can have it built with 11 tho.